frontend/src/utils/api/unilectivesApi.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { UnilectivesCourse } from 'types/unilectives';
+
+// eslint-disable-next-line import/prefer-default-export
+export const getCourseRating = async (code: string) => {
+  try {
+    const res = await axios.get(`https://unilectives.devsoc.app/api/v1/course/${code}`);
```

Comment:
knitpick: you can do `axios.get<return type>(...)` instead of using the `as` keyword

Reply:
Fixed